code/test/CoreTemplateStudio.Core.Test/Extensions/StringExtensionsTests.cs

```diff
@@ -10,11 +10,55 @@ namespace Microsoft.Templates.Core.Test.Extensions
     [Trait("ExecutionSet", "Minimum")]
     public class StringExtensionsTests
     {
+        [Theory]
+        [InlineData("string")]
+        [InlineData("string23TEST")]
```

Comment:
Here and below.
The very limited number of test cases is concerning.
Why no tests for whitespace? including at the start, middle, and end?
Why no tests for non-alphanumeric characters?
Why no tests with non-ascii characters?
Why no tests with strings containing non-printing characters?

Reply:
Hello @mrlacey !
Thanks a lot for all the feedback provided. We will be merging this PR as it brings some code changes that we need to test in WinTS. If you have any more feedback, we will resolve it on a different Pull Request.